src/rules/no-duplicate-load-rules/index.js

```diff
@@ -0,0 +1,184 @@
+"use strict";
+
+const { utils } = require("stylelint");
+const valueParser = require("postcss-value-parser");
+const getAtRuleParams = require("../../utils/getAtRuleParams");
+const namespace = require("../../utils/namespace");
+const isPlainObject = require("is-plain-object");
+const { isBoolean } = require("../../utils/validateTypes");
+const ruleUrl = require("../../utils/ruleUrl");
+
+const ruleName = namespace("no-duplicate-load-rules");
+const loadAtRules = ["import", "use"];
+
+const messages = utils.ruleMessages(ruleName, {
+  rejected: duplicate => `Unexpected duplicate load rule ${duplicate}`
+});
+
+const meta = {
+  url: ruleUrl(ruleName)
+};
+
+function rule(primary, secondaryOptions) {
+  return (root, result) => {
+    const validOptions = utils.validateOptions(
+      result,
+      ruleName,
+      {
+        actual: primary
+      },
+      {
+        actual: secondaryOptions,
+        possible: {
+          includeForward: isBoolean,
+          includeConfigurations: [isPlainObject.isPlainObject]
+        },
+        optional: true
+      }
+    );
+
+    if (!validOptions) {
+      return;
+    }
+
+    const imports = {};
+    const hasExplicitNamespace = new RegExp(
+      /\s+(as|with|show|hide)\s+\(?[^;]*\)?/g
+    );
+    if (secondaryOptions?.includeForward) {
+      loadAtRules.push("forward");
+    }
+
+    root.walkAtRules(atRule => {
+      if (!loadAtRules.includes(atRule.name.toLowerCase())) {
```

Comment:
`walkAtRules` accepts a regex as the first param, so we could possibly do this check there, but I'm not sure if it will be any cleaner (I guess it depends on if we'd want enable forward by default and something like `root.walkAtRules(/^(forward|import|use)$/i, atRule => {` )

Reply:
Thanks! Done.